### DISCUSSION TITLE
✅ Allow "Exclude Current" option in the Posts Module and the Loop Grid

### DISCUSSION THREAD

**Discussion by pixelloop** [elementor/elementor#22937] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

On most page builders I have used, like BeaverBuilder, when adding a loop grid at the bottom of a post, you can click a checkbox to "Exclude Current" so when it shows 4 more posts they can read, it doesn't show the post they are currently looking at which is of course redundant. 

Please consider including this in Elementor. 

This would make it a lot easier to include a posts module or loop grid at the end of the article template without it showing a post they are already looking at. 

### Describe the solution you'd like

A simple checkbox on the Posts module and Loop Grid module that when checked, excludes the post you are currently on. 

### Describe alternatives you've considered

Using a PHP query filter.

**Comment by jofbe** [comment]
I need this too please!

**Reply by Madmoigra** [reply]
Such an option already exists. Go to Exclude and exclude by -> Current post.
![image](https://github.com/elementor/elementor/assets/37309948/86ccda43-0e2b-4ad0-b032-239865a29ba1)

**Reply by pixelloop** [reply]
Did this exist back in June? I'm sure I checked this.
